evansmusomi/react-native-102
# Stores Place Info on Firebase

*What this does*
- Adds redux thunk to enable async network requests
- Saves shared place name and location on Firebase database

*How to verify*
- After sharing a place, a new place object is created in the Firebase database
- If hooked up to a react native debugger, you should see a response from Firebase with details of the object created
